### PR TITLE
Detailed description of Ursula's web status page

### PR DESCRIPTION
**Pull request by KPrasch**
**Type of PR:**
Documentation

**Required reviews:** 
2

**What this does:**
Introduces annotated screenshot of ursula's web status page with a legend containing wordy descriptions.

**Why it's needed:**
Reduce support questions workload through expanded documentation on node operation

**Notes for reviewers:**
Please focus on this section of the docs, feedback is welcomed on word choice/expansion:
Screenshot from 2021-04-26 12-12-39

### CODE REVIEW

File: docs/source/staking/running_a_worker.rst [hunk 1]
```diff
@@ -131,25 +135,24 @@ Initialize a new Worker
...
-    nucypher ursula init            \
-    --provider <PROVIDER URI>       \
-    --network <NETWORK NAME>        \
-    --signer <SIGNER URI>           \
+    -v ~/.ethereum/:/root/.ethereum               \
+    -p 9151:9151                                  \
+    -e NUCYPHER_KEYRING_PASSWORD                  \
+    nucypher/nucypher:latest                      \
+    nucypher ursula init                          \
+    --signer keystore:///root/.ethereum/keystore  \
```
Comment (fjarri) [inline docs/source/staking/running_a_worker.rst:146]: Not for this PR, but would it make sense to make that a default option?

File: docs/source/staking/running_a_worker.rst [hunk 2]
```diff
@@ -456,11 +459,34 @@ View logs for a CLI-launched or systemd Ursula:
...
+- *Client Version* - The version of nucypher this node is running
+- *Network Name* - The nucypher network name this node is running (mainnet, lynx, or ibex).
+- *Peer Count* - The total number of peers this node had discovered.
+- *Fleet State Checksum* - A checksum representing the currently known peer constituents and quantity
+- *Fleet State Icon* - A visual representation of the fleet state's checksum word and color
+- *Fleet State History* - The most recent historical fleet states known by this node
+- *Peer Nickname* - The nickname of a peer derived from it's staker address
+- *Peer Fleet State* - The current fleet state of a peer node
+- *Peer Staker Address* - The staker address of a peer
+- *Verified Nodes* - The collection of nodes that have been connected to and validated by this node
```
Comment (fjarri) [inline docs/source/staking/running_a_worker.rst:482]: I think it is possible for the node to be verified, but never have been learned from. I see such nodes on my status page.

**Review by fjarri — APPROVED** [review]

File: docs/source/staking/running_a_worker.rst (hunk 1 above)
Comment (cygnusv) [reply docs/source/staking/running_a_worker.rst:146]: It varies with the OS and with the ethereum network, so I wouldn't introduce a default

**Review by cygnusv — APPROVED** [review]

File: docs/source/staking/running_a_worker.rst (hunk 1 above)
Comment (fjarri) [reply docs/source/staking/running_a_worker.rst:146]: Both the OS and the network are know at the moment of running this command, so theoretically we could look into the default keystore path and see if there's actually a keystore in it.

File: docs/source/staking/running_a_worker.rst [hunk 3]
```diff
@@ -456,11 +459,34 @@ View logs for a CLI-launched or systemd Ursula:
...
+
+- *Nickname Icon* - A visual representation of the node's nickname words and colors
+- *Staker Nickname* - A nickname/codename for the node derived from the staker address
+- *Staker Address* - The Staker address this node is bonded to
+- *Client Version* - The version of nucypher this node is running
+- *Network Name* - The nucypher network name this node is running (mainnet, lynx, or ibex).
+- *Peer Count* - The total number of peers this node had discovered.
+- *Fleet State Checksum* - A checksum representing the currently known peer constituents and quantity
+- *Fleet State Icon* - A visual representation of the fleet state's checksum word and color
+- *Fleet State History* - The most recent historical fleet states known by this node
```
Comment (derekpierre) [inline docs/source/staking/running_a_worker.rst:478]:
```suggestion
- *Fleet State History* - The most recent historical fleet states known by this node, sorted from most recent to oldest
```

**Pull request merged**